json-rpc/spec.json

```diff
@@ -196,6 +196,101 @@
 				"$ref": "#/components/contentDescriptors/GasPrice"
 			}
 		},
+		{
+			"name": "eth_feeHistory",
+			"summary": "Transaction fee history",
+			"description": "Returns transaction base fee and effective priority fee history for the requested/supported block range.",
+			"params": [
+				{
+					"name": "blockCount",
+					"description": "Number of blocks in the requested range.",
+					"required": true,
+					"schema": {
+						"$ref": "#/components/schemas/Integer"
+					}
+				},
+				{
+					"name": "lastBlock",
+					"description": "Last block number of the requested range.",
+					"required": true,
+					"schema": {
+						"oneOf": [
+							{
+								"$ref": "#/components/schemas/BlockNumber"
+							},
+							{
+								"$ref": "#/components/schemas/BlockNumberTag"
+							}
+						]
+					}
+				},
+				{
+					"name": "rewardPercentiles",
+					"description": "A list of percentile values (0-100) to sample from each block's miner rewards in ascending order, weighted by gas used.",
+					"schema": {
+						"title": "rewardPercentiles",
+						"type": "array",
+						"items": {
+							"type": "number"
+						}
+					}
+				}
+			],
+			"result": {
+				"name": "feeHistoryResult",
+				"description": "Fee history for the returned block range (the intersection of requested and supported range).",
+				"schema": {
+					"title": "feeHistoryResults",
+					"description": "Fee history results.",
+					"type": "object",
+					"required": [
+						"firstBlock",
+						"baseFee",
+						"gasUsedRatio"
+					],
+					"properties": {
+						"firstBlock": {
+							"title": "firstBlock",
+							"description": "First block number of the returned range.",
+							"$ref": "#/components/schemas/BlockNumber"
+						},
+						"baseFee": {
+							"title": "baseFeeArray",
+							"description": "An array of block base fees (including the next block after the returned range).",
+							"type": "array",
+							"items": {
+								"$ref": "#/components/schemas/Integer"
+							}
+						},
+						"gasUsedRatio": {
+							"title": "gasUsedArray",
+							"description": "An array of block gas used ratios.",
```

Comment:
Is this a decimal number between 0 and 1?  Might be good to provide a bit more description here.

Reply:
done